docs/docs-content/clusters/data-center/nutanix/create-manage-nutanix-cluster.md

```diff
@@ -0,0 +1,146 @@
+
+  #### Master Pool 
+
+  | **Variable** | **Description** |
+  |--------------|-----------------|
+  | `${CLUSTER_NAME}`| The name of the Nutanix workload cluster. |
+  | `${CONTROL_PLANE_ENDPOINT_IP}`| The host static IP address. |
+  | `${NUTANIX_SSH_AUTHORIZED_KEY}`| Provide your SSH key. |
+  | `${KUBERNETES_VERSION}`| Specify the Kubernetes version for your cluster, and precede the version number with  'v'. For example: v.1.26.3 |
+  | `${NUTANIX_PRISM_ELEMENT_CLUSTER_NAME}`| The name of your Nutanix Prism cluster. |
+  | `${NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME}` | The name of your OS image. |
+  | `${NUTANIX_SUBNET_NAME}` | The name of the subnet for your Nutanix Prism cluster. |
+
+
+<!-- ${CLUSTER_NAME}
+${CONTROL_PLANE_ENDPOINT_IP}
+${NUTANIX_SSH_AUTHORIZED_KEY}
+${KUBERNETES_VERSION} - precede with v
+${NUTANIX_PRISM_ELEMENT_CLUSTER_NAME}
+${NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME} - replace with OS image
+${NUTANIX_SUBNET_NAME}  -->
+
+  #### Worker-Pool
+
+  | **Variable** | **Description** |
+  |--------------|-----------------|
+  | `${NUTANIX_SSH_AUTHORIZED_KEY}`| Provide your SSH key. |
+  | `${KUBERNETES_VERSION}`| Specify the Kubernetes version for your cluster, and precede the version number with  'v'. For example: v.1.26.3 |
+  | `${NUTANIX_PRISM_ELEMENT_CLUSTER_NAME}`| The name of your Nutanix Prism cluster. |
+  | `${NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME}` | The name of your OS image. |
+  | `${NUTANIX_SUBNET_NAME}` | The name of the subnet for your Nutanix Prism cluster. |
+
+
+<!-- ${NUTANIX_SSH_AUTHORIZED_KEY}
+${KUBERNETES_VERSION} - precede with v, no quotes 
+${NUTANIX_PRISM_ELEMENT_CLUSTER_NAME}
+${NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME}
+${NUTANIX_SUBNET_NAME}  -->
+
+
+10. Click **Next** when you are done.
+
+11. Review the options for OS Patchine Schedule, scanning, backups, and RBAC. 
+
+12. Click **Validate** and review the cluster configuration and setting settings summary.
+
+13. Click **Finish Configuration** to deploy the cluster. The cluster details page contains the status and details of the deployment. Use this page to track the deployment progress. Provisioning clusters can take several minutes to complete.
+
+14. To edit node pool configurations, from the cluster details page, click the **Nodes** tab and select the node pool you wnat to edit, and click the **Edit** button. The YAML displays. 
+
+15. To edit cluster settings, from the cluster details page, click the **Settings** button and select **Cluster Configuration**. Edit the YAML that Palette displays.
+
+
+## Validate
+
+1.  Log in to [Palette](https://console.spectrocloud.com/).
+
+2. Navigate to the **left Main Menu** and select **Clusters**. The Clusters page displays a list of all available clusters that Palette manages.
+
+3. Click on the cluster you created to view its details page.
+
+4. Ensure the **Cluster Status** field displays **Running**.
+
+
+## Resources
```

Comment:
Do we need this section here?

Reply:
Removed this placeholder section, as it's not needed.